server/src/main/resources/cloudFoundry/deployment_scripts/configureMysql.py

```diff
@@ -0,0 +1,47 @@
+import mysql.connector
+import ast
+import sys
+
+# needs a special python mysql connector installation
+def main():
+    mysqlConfigFile = 'mysqlConfig.txt'
+    # should be a sql file
+    strConfigureFile = str(sys.argv[1])
+
+    if ".sql" not in strConfigureFile[len(strConfigureFile)-4:]:
+        print("needs an .sql File for db init")
+        exit(1)
+
+    try:
+        with open(strConfigureFile, 'r') as dbinit_file:
+            dbinit_command = dbinit_file.read()
+    except IOError as err:
+        print("Failed to read dbinit File named %s. Where it is?" %(strConfigureFile))
```

Comment:
consider using `print("Failed to read dbinit File named {}. Where it is?".format(strConfigureFile))`